source/unified-test-format/tests/invalid/expectedCommandEvent-commandFailedEvent-hasServerId-type.yml

```diff
@@ -0,0 +1,16 @@
+description: expectedCommandEvent-commandFailedEvent-hasServerId-type
+
+schemaVersion: '1.3'
+
+createEntities:
+  - client:
+      id: &client0 client0
+
+tests:
+  - description: invalid type for commandFailedEvent hasServerId field
```

Comment:
No objection to leaving this as-is, but you can use "foo" (or similar filler) for the "invalid" test descriptions since these will never be executed by a test runner.

Reply:
Changed to `foo` here and in other files for consistency.